README.md

```diff
@@ -0,0 +1,5 @@
+# NeoSearchHelper
+Adds search icons under items in various areas.
+Almost Abandoned Attic is tricky because the image and text are linked to each other in the No Haggle option. To use the Search Helper icons in the Attic, hit "Cancel" and the link will open.
+
+![image](https://github.com/themeels/NeoSearchHelper/assets/64810580/3a554a86-1dd7-47df-bf68-6aee02ee1984)
```

Comment:
If we're going to be adding a readme image to this, the image should be hosted as an asset in the git repository directly, or as a public image like on imgur.

Reply:
Not 100% sure how one would add it as an hosted asset to the git repository, would it just need to be added by Dice?